specification/components/schemas/responses/CreateReferralErrorOutcome.yaml

```diff
@@ -34,6 +34,6 @@ headers:
 content:
   application/fhir+json:
     schema:
-      $ref: '../eRS-OperationOutcome.yaml'
+      $ref: '../STU3-OperationOutcome.yaml'
```

Comment:
Renaming this will have to be implemented on the new set of provider side endpoints (when they merge in). Take care when rebasing (those endpoints aren't in develop yet)